### PR TITLE
Common tools

### PR DESCRIPTION
**Pull request by EmmaRenauld**
You can review the text parts (descriptions of the tools).

The code parts (how to install the tools) are based on my notes from september but will probably be updated when I receive my new computer and start again installing everything.

### CODE REVIEW

**Comment by EmmaRenauld** [comment]
Max and Chantal would like to see this if you have some time to review!  @arnaudbore   @GuillaumeTh  @jchoude

File: source/tools/amico.rst [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Amico / Commit
+==============
+
+To Do. Can you could help me?
```
Comment (GuillaumeTh) [inline source/tools/amico.rst]: @arnaudbore do you have a description or ideas to what we could say about amico/commit ?

File: source/tools/ants.rst [hunk 2]
```diff
@@ -1,4 +1,41 @@
...
+
+    .. code-block:: bash
+
+        mkdir ANTs
+        cd ANTs
+        git clone https://github.com/stnava/ANTs.git code
+        mkdir build
+        cd build
+        ccmake ../code # Press c twice and then g
+        make -j 8
```
Comment (GuillaumeTh) [inline source/tools/ants.rst]: Maybe add a comment to explain that 8 is the number of threads

File: source/tools/ants.rst [hunk 3]
```diff
@@ -1,4 +1,41 @@
...
+        cp ../code/Scripts/*.sh bin/
+
+    Then add these lines in your .bashrc:
+
+    - From a terminal: :bash:`sudo gedit ~/.bashrc`
```
Comment (GuillaumeTh) [inline source/tools/ants.rst]: You don't need to have sudo right to edit bashrc

File: source/tools/ants.rst [hunk 4]
```diff
@@ -1,4 +1,41 @@
...
+    - Copy this in your file:
+
+        .. code-block:: bash
+
+            export PATH=~/Code/ANTs/build/bin:${PATH}
```
Comment (GuillaumeTh) [inline source/tools/ants.rst]: following the previous commands it is `code` and not `Code`

File: source/tools/environments.rst [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+    .. code-block:: bash
+
+        sudo pip install virtualenvwrapper
+        mkdir ~/Envs
+        NAME=somename # Ex: $NOM could be my_env_py3
+                      # depending on the use you will give to this environment.
+                      # Try to be explicit
+        # One of:
+        mkvirtualenv $NAME
+        mkvirtualenv --python=python3.7 $NAME   # To use one precise python version
```
Comment (GuillaumeTh) [inline source/tools/environments.rst]: Are you sure this line works ? I think --python take only a path

File: source/tools/fsl_freesurfer.rst [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+FSL
+---
+
+*What is it?*
+
+    FSL is a set of tools to analyse FMRI, MRI and DTI brain imaging data. Here is the link to their `wiki page <https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/>`_.
+
+    In the lab, we mainly use `Bet <https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/BET>`_ (for skullstripping).
+
+    Previously, we also used some of these tools, but now we prefer FreeSurfer or Ants: `FAST <https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FAST>`_ (to segment a T1 into GM, WM, CSF), `FIRST <https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FIRST>`_ (for subcortical GM segmentation), and `FLIRT <https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FLIRT>`_/`FNIRT <https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FNIRT>`_ (for linear/non-linear registration).
```
Comment (GuillaumeTh) [inline source/tools/fsl_freesurfer.rst]: We use FAST a lot. So you can move FAST in the previous sentence

File: source/tools/fsl_freesurfer.rst [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+*How to install it?*
+
+    To install, you must download fslinstaller.py at https://fsl.fmrib.ox.ac.uk/fsldownloads_registration. You will have to create an account.
+
+    Then from a terminal: :bash:`python fslinstaller.py`.
```
Comment (GuillaumeTh) [inline source/tools/fsl_freesurfer.rst]: Maybe add that fslinstaller works only with python2.7

File: source/tools/mi-brain.rst [hunk 8]
```diff
@@ -1,4 +1,14 @@
...
 Mi-Brain
-==============
+========
+
+`Mi-Brain <https://www.imeka.ca/fr/mi-brain/>`_ is a GUI viewer developped by IMEKA (a Sherbrooke compagny for diffusion analysis co-founded by Maxime Descoteaux and Pierre-Marc Jodoin.
+
+To install it, scroll down to the Q/R section and follow the instructions.
+
+
+\* Other useful viewers: mrview (see MRtrix), `tracVis <http://trackvis.org/download/>`_.
```
Comment (GuillaumeTh) [inline source/tools/mi-brain.rst]: `tracVis` -> `trackvis`

File: source/tools/mrtrix.rst [hunk 9]
```diff
@@ -1,4 +1,42 @@
...
+*How to install it?*
+
+    In a terminal, from the directory where you want to install it:
+
+    .. code-block:: bash
+
+        git clone https://github.com/MRtrix3/mrtrix3.git
+        cd mrtrix3
+        ./configure
+        ./build
```
Comment (GuillaumeTh) [inline source/tools/mrtrix.rst]: You can add `NUMBER_OF_PROCESSORS=8` before ./build to set the number of threads to use to build mrtrix

File: source/tools/scilpy_dipy_tractoflow.rst [hunk 10]
```diff
@@ -0,0 +1,70 @@
...
+
+    .. code-block:: bash
+
+        scil_compute_fodf.py -h
+
+*How to install it*
+
+    Users: you can follow the instructions on the Github page. Clone the repository, install the requirements and install through the setup.py as explained.
+
+    VÉRIFIER QUE MA PR POUR LA DESCRIPTION POUR LES DÉVELOPPERS EST AJOUTÉE DANS SCILPY.
```
Comment (GuillaumeTh) [inline source/tools/scilpy_dipy_tractoflow.rst]: Please remove this line. I will merge this PR soon.

File: source/tools/environments.rst (hunk 5 above)
Comment (EmmaRenauld) [reply source/tools/environments.rst]: I just checked and it works!

**Comment by EmmaRenauld** [comment]
Tell me if you want me to rebase on the master.

**Comment by GuillaumeTh** [comment]
Yes @EmmaRenauld can you rebase please

**Pull request merged**